app/helpers/collecting_events_helper.rb

```diff
@@ -100,7 +100,7 @@ def collecting_event_dates_tag(collecting_event)
   end
 
   def collecting_event_verbatim_locality_tag(collecting_event)
-    return nil unless !(collecting_event&.verbatim_locality&.blank?)
+    return nil unless collecting_event&.verbatim_locality&.present?
```

Comment:
Last safe navigation operator is technically not required (only on Rails):
```ruby
$ rails c
Loading development environment (Rails 6.0.4)
irb: warn: can't alias context from irb_context.
2.7.4 :001 > nil.present?
 => false 
2.7.4 :002 > nil&.present?
 => nil 
```

@mjy can you confirm it is safe for this helper to return nil (given it wasn't able to previously)?

Reply:
The only place it's called is here https://github.com/SpeciesFileGroup/taxonworks/blob/19853faef939f400e93b17919e5d3aa8a2b177e3/app/helpers/collecting_events_helper.rb#L14-L26

Since `compact` is called on it I can't imagine it would cause any issues.